static/app/bootstrap/processInitQueue.tsx

```diff
@@ -0,0 +1,56 @@
+/**
+ * This allows server templates to push "tasks" to be run after application has initialized.
+ * The global `window.__onSentryInit` is used for this.
+ */
+export async function processInitQueue() {
+  if (!Array.isArray(window.__onSentryInit)) {
+    return;
+  }
+
+  await Promise.all(
```

Comment:
You can just return this promise and drop the async from the function

Reply:
eslint starts to complain about inconsistent returns - I think it's fine to return `Promise<void>` vs `Promise<void[]>` (this assumes we'll never need the result of the promise, which hopefully we won't as these are more legacy templates).

Reply:
Ah yeah got you